### PR TITLE
Refine README and demo

### PR DESCRIPTION
**Pull request by ishkawa**
- Fix instructions to install
- Not create API subclass in README and demo
- Add tips about Himotoki

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -336,6 +330,70 @@ struct SomePaginatedRequest: Request {
 }
 ```
 
+### Combining with [Himotoki](https://github.com/ikesyo/Himotoki)
```
Comment (ikesyo) [inline README.md]: How about "Combination with"?

File: README.md [hunk 2]
```diff
@@ -295,11 +289,11 @@ SomeAPI.sendRequest(request) { result in
 
 ```swift
 struct PaginatedResponse<T> {
-    var results: Array<T>
+    var results: [T]
```
Comment (ikesyo) [inline README.md]: :+1:

File: README.md (hunk 1 above)
Comment (ikesyo) [inline README.md]: Would it be OK to add the link below, instead of linking within a header?

**Comment by ikesyo** [comment]
Seems legit! :sparkles: And also thank you for referring to my library!! :wink:

File: README.md (hunk 1 above)
Comment (ishkawa) [inline README.md]: OK, I'll fix it!

**Pull request merged**